Tasks/KubernetesManifestV0/Strings/resources.resjson/en-US/resources.resjson

```diff
@@ -91,5 +91,6 @@
   "loc.messages.InvalidBaselineAndCanaryReplicas": "Invalid value for replica count.",
   "loc.messages.InvalidTimeoutValue": "Invalid value for timeout. Enter a valid number.",
   "loc.messages.RolloutStatusTimedout": "Rollout status check failed.",
-  "loc.messages.EnvironmentLink": "For more information, go to %s"
+  "loc.messages.EnvironmentLink": "For more information, go to %s",
+  "loc.messages.NullContainerImage": "Container image is null."
```

Comment:
Could you extract the resource name and type and append in this log to aid troubleshooting.
Something like `Container image is null in resource: '{0}' of type:  '{1}'`